CONTRIBUTING.md

```diff
@@ -130,11 +133,30 @@ func (ssa *SQLStorageAuthority) AddPerson(p Person) (error) {
 }
 ```
 
-Add a migration with
+You will also need to update the `initTables` function from `sa/database.go` to
+tell Gorp which table to use for your versioned model structs. Make sure to
+consult the flag you defined so that only **one** of the table maps is added at
+any given time, otherwise Gorp will error.  Depending on your table you may also
```

Comment:
@rolandshoemaker Yes, I tried adding two types with the same table name and it caused inserts to fail.